db/schema.rb

```diff
@@ -57,7 +57,7 @@
     t.bigint "vendor_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "license_type"
+    t.string "type"
```

Comment:
@larshoubak what was the purpose of this renaming? "type" cannot be used as a name for a column when querying. Also if we rename I need to rewrite a lot of my code...